src/daemon/collectd.h

```diff
@@ -227,7 +227,7 @@
 #endif
 
 #ifndef COLLECTD_DEFAULT_INTERVAL
-#define COLLECTD_DEFAULT_INTERVAL 60.0
+#define COLLECTD_DEFAULT_INTERVAL 10.0
```

Comment:
Please update the description to include that the default interval is being updated to 10 seconds.

Reply:
That's just the default collectd interval. Our packaged config still uses 60s. I figured putting all that in the PR description would have been somewhat misleading... WDYT?

Reply:
I would suggest removing this change, if you don't intend to communicate it in the PR description.

Reply:
Removing it would not make this a clean revert. Added to the PR description instead. PTAL.